topics/metabolomics/tutorials/gcms/tutorial.md

```diff
@@ -0,0 +1,432 @@
+--- 
+layout : tutorial_hands_on
+enable: false
+
+title : 'Mass spectrometry : GC-MS analysis with metaMS package'
+level : Introductory
+zenodo_link : 'https://zenodo.org/record/3244991' 
+questions : 
+    - What are the main steps of GC-MS datas processing for metabolomic analysis ? 
+    - How te be able to annotate the maximum of unknowns using Galaxy ? 
+objectives : 
+    - To be sure you have already comprehend the diversity of MS pre-processing analysis. 
+    - To learn the principal functions of metaMS package through Galaxy.
+    - To evaluate the potential of this new GC-MS workflow for GC-MS metabolomic analysis. 
+time_estimation : 2H 
+key_points : 
+    - Have a good file containing all your peaks during the first stopover 
+    - Find all your unknowns in your datas
+    - Find your stanards if you have some 
+requirements :
+  - type: "internal"
+    topic_name: metabolomics
+    tutorials: 
+      - lcms
+contributors : 
+    - jsaintvanne
+
+--- 
+
+# Introduction
+{:.no_toc}
+
+A lot of packages are available for the analysis of GC-MS or LC-MS datas. Typically, hardware vendors provide software that is optimized for the instrument and allow a direct interaction of the lab scientist with the data. Some other open-source alternatives such as **XCMS** are also able to be integrated easily in web interfaces, allowing large numbers of files to be processed simultaneously. Because of the generality of packages like **XCMS**, several other packages have been developped to use the functionality of **XCMS** for optimal performance in a particular context. Package **metaMS** does so for the field of untargeted metabolomics, focuses on the GC-MS analysis during this tutorial. One of the goals **metaMS** was to set up a simple system with few user-settable parameters, capable of handling the vast majority of untargeted metabolomics experiments. During this tutorial, we will learn how to process easily a test dataset from raw files to the annotation using W4M Galaxy. Datas are from {% cite Dittami2012 %} and have been used as test dataset for the development of the Galaxy wrappers. 
```

Comment:
```suggestion
A lot of packages are available for the analysis of GC-MS or LC-MS data. Typically, hardware vendors provide software that is optimized for the instrument and allow a direct interaction of the lab scientist with the data. Some other open-source alternatives such as **XCMS** are also able to be integrated easily in web interfaces, allowing large numbers of files to be processed simultaneously. Because of the generality of packages like **XCMS**, several other packages have been developed to use the functionality of **XCMS** for optimal performance in a particular context. Package **metaMS** does so for the field of untargeted metabolomics, focuses on the GC-MS analysis during this tutorial. One of the goals **metaMS** was to set up a simple system with few user-settable parameters, capable of handling the vast majority of untargeted metabolomics experiments. 

During this tutorial, we will learn how to process easily a test dataset from raw files to the annotation using W4M Galaxy. Datas are from {% cite Dittami2012 %} and have been used as test dataset for the development of the Galaxy wrappers. 
```